testing/resources/golang_versioned_hooks_repo/go.mod

```diff
@@ -0,0 +1,5 @@
+module golang-hello-world
+
+go 1.18
+
+require github.com/BurntSushi/toml v1.1.0
```

Comment:
you can probably take the existing local go test and set `language_version` -- that'd be simpler than a whole separate repo

Reply:
done, now we only have one repo -- I have added a little arg in the `main.go`  that helps tell when running Go system, I'm not sure if it's sustainable.